test/08_api-keys.ts

```diff
@@ -88,13 +93,18 @@ describe('create provisioning apikey', function () {
 						key: provisioningKey,
 					},
 					options: {
-						$select: 'name',
+						$select: ['name', 'description'],
 					},
 				});
 
 				expect(apiKeyResp).to.have.property('name');
 				expect(apiKeyResp?.name).to.be.a('string');
 				expect(apiKeyResp?.name).to.equal(`provision-key-${applicationId}`);
+				expect(apiKeyResp).to.have.property('description');
+				expect(apiKeyResp?.description).to.be.a('string');
+				expect(apiKeyResp?.description).to.equal(
+					`Sample key for application-${applicationId} description.`,
+				);
```

Comment:
This can just be:
```suggestion
				expect(apiKeyResp).to.have.property('description', `Sample key for application-${applicationId} description.`);
```